src/components/forms/Form/ComplexForm.stories.tsx

```diff
@@ -278,78 +270,71 @@ const Template: StoryFn<typeof Form> = (args) => {
         <Field name="password">
           <PasswordInput label="Password field" />
         </Field>
-        <Field
-          name={['select', 'one']}
-          label="Select field"
-          tooltip="Additional field description"
-        >
-          <Select>
-            <Item key="one">One</Item>
-            <Item key="two">Two</Item>
-            <Item key="three">Three</Item>
-          </Select>
-        </Field>
-        <Field name="combobox" label="ComboBox field">
-          <ComboBox>
-            <Item key="one">One</Item>
-            <Item key="two">Two</Item>
-            <Item key="three">Three</Item>
-          </ComboBox>
-        </Field>
-        <Field isLoading name="combobox2" label="ComboBox Loading field">
-          <ComboBox>
-            <Item key="one">One</Item>
-            <Item key="two">Two</Item>
-            <Item key="three">Three</Item>
-          </ComboBox>
-        </Field>
-        <Field
-          name="checkboxGroup"
-          label="Checkbox group"
-          rules={[
-            {
-              required: true,
-              message: 'Specify at least a single option',
-            },
-          ]}
-        >
-          <CheckboxGroup orientation="horizontal">
-            <Checkbox value="one">One</Checkbox>
-            <Checkbox value="two">Two</Checkbox>
-            <Checkbox value="three">Three</Checkbox>
-          </CheckboxGroup>
-        </Field>
-        <Field name="radioGroup" label="Radio group">
-          <Radio.Group>
-            <Radio value="one">One</Radio>
-            <Radio value="two">Two</Radio>
-            <Radio value="three">Three</Radio>
-          </Radio.Group>
-        </Field>
-        <Field
-          name="checkbox"
-          rules={[{ required: true, message: 'This field is required' }]}
-        >
-          <Checkbox label="Checkbox field" />
-        </Field>
-        <Field
-          name="switch"
-          rules={[{ required: true, message: 'This field is required' }]}
-        >
-          <Switch label="Switch field" />
-        </Field>
-        <Field
-          name="number"
-          rules={[{ required: true, message: 'This field is required' }]}
-        >
-          <NumberInput label="Number field" minValue={-1} />
-        </Field>
-        <Field
-          name="slider"
-          rules={[{ required: true, message: 'This field is required' }]}
-        >
-          <RangeSlider showInput minValue={0} maxValue={100} />
-        </Field>
+        {/*<Field*/}
```

Comment:
Why did we comment out these lines?

Reply:
Performance issues. In the storybook it takes too long to insert all the styles for the story.
Before that PR, we have more atomic styles for buttons. So it took less time to insert them. Not much of a difference, but it crossed the 15s limit in chromatic.
I don't want to remove this code so that I can get back to it for testing purposes. I don't see any other options.

Reply:
Does it mean that this perf regression affects cloud perf as well?

Reply:
In my opinion, if a change causes perf regression, we should either dismiss that change or fix perf issues

Reply:
Yes, it causes, on the first load. But the following renderings will be faster because many used styles are already prepared.
The ComplexForm story was slow before. But now it has surpassed the limit.
I tested Cube Cloud. There is no measurable effect of this change. But after the release, I will have to migrate all custom buttons on Cloud to the standard ones.